code/modules/mob/living/carbon/human/species_types/modder.dm

```diff
@@ -0,0 +1,90 @@
+//gene modded humans
+/datum/species/human/modder
+	name = "Modded Human"
+	id = SPECIES_MODDER
+	//say_mod = "meows"
+
+	mutant_bodyparts = list("tail_human" = "Cat", "ears" = "Cat", "wings" = "None")
+
+	mutantears = /obj/item/organ/ears/cat/modder
+	mutant_organs = list(/obj/item/organ/tail/cat/modder)
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
+	//species_language_holder = /datum/language_holder/felinid
+	payday_modifier = 0.75
+	ass_image = 'icons/ass/asscat.png'
+	examine_limb_id = SPECIES_HUMAN
+
+//Curiosity killed the cat's wagging tail.
+/datum/species/human/modder/spec_death(gibbed, mob/living/carbon/human/H)
+	if(H)
+		stop_wagging_tail(H)
+
+/datum/species/human/modder/spec_stun(mob/living/carbon/human/H,amount)
+	if(H)
+		stop_wagging_tail(H)
+	. = ..()
+
+/datum/species/human/modder/can_wag_tail(mob/living/carbon/human/H)
+	return mutant_bodyparts["tail_human"] || mutant_bodyparts["waggingtail_human"]
+
+/datum/species/human/modder/is_wagging_tail(mob/living/carbon/human/H)
+	return mutant_bodyparts["waggingtail_human"]
+
+/datum/species/human/modder/start_wagging_tail(mob/living/carbon/human/H)
+	if(mutant_bodyparts["tail_human"])
+		mutant_bodyparts["waggingtail_human"] = mutant_bodyparts["tail_human"]
+		mutant_bodyparts -= "tail_human"
+	H.update_body()
+
+/datum/species/human/modder/stop_wagging_tail(mob/living/carbon/human/H)
+	if(mutant_bodyparts["waggingtail_human"])
+		mutant_bodyparts["tail_human"] = mutant_bodyparts["waggingtail_human"]
+		mutant_bodyparts -= "waggingtail_human"
+	H.update_body()
+
+/datum/species/human/modder/on_species_gain(mob/living/carbon/C, datum/species/old_species, pref_load)
+	if(ishuman(C))
+		var/mob/living/carbon/human/H = C
+		if(H.dna.features["ears"] != "None")
+			var/obj/item/organ/ears/cat/modder/ears = new
+			ears.Insert(H, drop_if_replaced = FALSE)
+		else
+			mutantears = /obj/item/organ/ears
+		if(H.dna.features["tail_human"] != "None")
+			var/obj/item/organ/tail/cat/modder/tail = new
+			tail.Insert(H, special = TRUE, drop_if_replaced = FALSE)
+		else
+			mutant_organs = list()
+	return ..()
+
+/datum/species/human/modder/prepare_human_for_preview(mob/living/carbon/human/human)
+	human.hairstyle = "Hime Cut"
+	human.hair_color = "#ffcccc" // pink
+	human.update_hair()
+
+	var/obj/item/organ/ears/cat/cat_ears = human.getorgan(/obj/item/organ/ears/cat)
+	if (cat_ears)
+		cat_ears.color = human.hair_color
+		human.update_body()
+
+/datum/species/human/modder/get_species_description()
+	return "Modded humans are humans who have undergone genetic modifcation \
+		for various reasons. Perhaps this was for animal ears, changing \
+		height, or maybe even to increase penis size! Who knows!"
```

Comment:
This is funny, but questionable.

Reply:
yeah i was hoping someone would actually write somewhat reasonable shit